exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/text_ai/utils/txaie_extraction_wrapper.ipynb

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.                language_alias = "PYTHON3_TXAIE" #conf.get(CKey.language_alias)todo change once var exists in sectret store
</pre><p>This on is not used, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>